### PR TITLE
Feature/user class

### PR DESCRIPTION
**Pull request by st0nebreaker**
#### What's this PR do?
User class for data structuring set up, takes in one User object. Holds all the properties of the User. And returns first name when User.findName() is run.

#### Where should the reviewer start?
Files changed tab on GH. Preview files ceated.

#### How should this be manually tested?
Locally, pull down and run npm, make sure tests look good & pass.

#### Any background context you want to provide?
Simple data structure setup.

#### What are the relevant tickets?
Trello

#### Questions:

### CODE REVIEW

File: test/User-class-test.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+const expect = chai.expect;
+
+const User = require('../src/User-class');
+const userData = require('../data/users');
+
+describe('User', () => {
+
+	let user;
+
+	beforeEach(() => {
```
Comment (carlymclift) [inline test/User-class-test.js:11]: I didn't know we could do beforeEach like this! Good to know! I'll do the same on my tests.

File: src/User-class.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class User {
```
Comment (carlymclift) [inline src/User-class.js:1]: Looks great!

File: .eslintrc [hunk 3]
```diff
@@ -14,7 +14,7 @@
     "comma-spacing": ["warn", { "before": false, "after": true }],
     "curly": "error",
     "semi-spacing": ["error", { "before": false, "after": true }],
-    "indent": ["warn", 2],
+    // "indent": ["warn", 2],
```
Comment (carlymclift) [inline .eslintrc:17]: Did we need to comment this line out?

**Review by carlymclift — APPROVED** [review]
Awesome! Looks great, I'll model my testing style after this so that it's consistent and it looks really good (like add beforeEach and do ES6)

File: test/User-class-test.js (hunk 1 above)
Comment (st0nebreaker) [reply test/User-class-test.js:11]: Yea Hannah told me from the last project! My first time trying it :)

File: .eslintrc (hunk 3 above)
Comment (st0nebreaker) [reply .eslintrc:17]: Oops my mistake. I commented it out because the lines were annoying me on VS Code haha. Sorry about that.

**Pull request merged**